components/base/sensorcontrolled/sensorcontrolled_test.go

```diff
@@ -511,3 +512,26 @@ func TestSensorBaseMoveStraight(t *testing.T) {
 		orientationValue = defaultOrientationValue
 	})
 }
+
+func TestSensorBaseDoCommand(t *testing.T) {
+	ctx := context.Background()
+	logger := logging.NewTestLogger(t)
+	deps, cfg := msDependencies(t, []string{"setvel1", "position1", "orientation1"})
+	b, err := createSensorBase(ctx, deps, cfg, logger)
+	test.That(t, err, test.ShouldBeNil)
+
+	sb, ok := b.(*sensorBase)
+	test.That(t, ok, test.ShouldBeTrue)
```

Comment:
I would be surprised if you need this, the resource creator function should include the DoCommand in the struct returned.

Reply:
did this in order to set the tunedVals on the sensorBase struct

Reply:
You can pass them in through the config in the creator?

Reply:
sure

Reply:
actually no because if there's non-zero values passed into the config the tuned values returned from DoCommand will be empty. it _only_ returns tuned values so they need to be set after the base is created

Reply:
ooooh, interesting; okay, a good end-to-end test would be running the tuning loop with mocks. For another pr!

However, for this pr, if I've already tuned my PID, copied it over to my config, and want to grab those values programmatically because the semantics of this command tell me I can... that's not a great flow. We'll think about this again sometime.